pootle_pytest/fixtures/site.py

```diff
@@ -20,9 +20,16 @@ def site_matrix(request, system, settings):
         TranslationProjectFactory, StoreFactory, UnitFactory)
 
     from pootle_store.models import UNTRANSLATED, TRANSLATED, FUZZY, OBSOLETE
+    from pootle_app.models import Directory
 
-    # create root and projects directories
-    DirectoryFactory(name="projects", parent=DirectoryFactory(parent=None))
+    # create root and projects directories, first clear the class cache
```

Comment:
I apologize if my comments come across as rude, that's definitely not the intention. When I go through many PR/issues at a time it's true that I tend to write short and straight to the point as @unho says. This includes the risk of not knowing how to express things properly, which opens up the gap for misunderstandings (English is my third language).

Having said that, I'll do my best to avoid potential conflicts as this in the future, and I'm happy to take this conversation to private.